util/csvresponse.py

```diff
@@ -137,6 +137,13 @@ def _create_csv(self, dataset, filehandle):
         # filter the dataset
         dataset = dataset.drop(drop.intersection(dataset))
 
+        # rename the parameter without the stream_name prefix (12544 AC1)
```

Comment:
for csv's and json they don't have the extra metadata for each variables that netcdf has to identifies the actual stream from which the parameter came from.  netcdf can get away with removing the stream name from the variable because it has additional metadata attributes that help identify the true source of the data.

removing the stream name introduces a change in the interface of the data which may break user developed m2m tools.  before implementing this change for csv's and json we probably should get feedback from the user community if this is what they want.

Reply:
Agreed. Will pull the name changes and update the ticket to request input/direction from the user community.

Reply:
I think this will also correct the test failure.